Makefile

```
@@ -90,8 +94,19 @@ kcm-dist-release: helm yq
 templates-generate:
 	@hack/templates.sh
 
+CAPO_DIR := $(PROVIDER_TEMPLATES_DIR)/cluster-api-provider-openstack
+CAPO_ORC_VERSION ?= v$(shell grep 'orcRelease:' $(CAPO_DIR)/values.yaml | cut -d '"' -f 2)
+CAPO_ORC_FILE := "$(CAPO_DIR)/files/orc-$(shell echo $(CAPO_ORC_VERSION) | sed 's/\./\-/g' | cut -c2-).yaml"
+.PHONY: capo-orc-fetch
+capo-orc-fetch:
+	@if [ -f "$(CAPO_ORC_FILE)" ]; then \
+  		echo "ORC installation file with version $(CAPO_ORC_VERSION) already exists in $(CAPO_DIR) chart."; \
+  	else \
+		curl -L --fail -s https://github.com/k-orc/openstack-resource-controller/releases/download/$(CAPO_ORC_VERSION)/install.yaml -o $(CAPO_ORC_FILE); \
+	fi
```

```suggestion
CAPO_ORC_FILE := "$(CAPO_DIR)/files/orc-$(shell echo $(CAPO_ORC_VERSION) | sed 's/\./-/g' | cut -c2-).yaml"
.PHONY: capo-orc-fetch
capo-orc-fetch:
	@test -s "$(CAPO_ORC_FILE)" || curl -L --fail -s https://github.com/k-orc/openstack-resource-controller/releases/download/$(CAPO_ORC_VERSION)/install.yaml -o $(CAPO_ORC_FILE)
```

might be simplified (no need in echo; no need to escape `-` sym)